src/components/sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,64 @@
+const Sidebar = () => {
+    return (
+        <aside className="px-4 py-1 my-2">
+            <div className="aside-heading">
+                <h5>Filters</h5>
+                <button className="button-link">Clear</button>
+            </div>
+            <hr />
+            <h6 className="my-1">Category</h6>
+            <ul className="side-nav font-medium pl-1">
+                <li>
+                    <input id="category-1" type="checkbox" />
```

Comment:
Suggestion: 
Can have a checkbox component and reuse the same.
Same can been done for radio btn components.